README.md

```diff
@@ -32,3 +32,27 @@ are affected by that command:
 
 
 All the added files should be recognized by Android Studio.
+
+## Building an apk
+
+* Ensure you have Android Studio and gradle installed
+
+On macOS you can do:
+
+```
+brew cask install android-studio
+brew install gradle
```

Comment:
I think I did not need to install `gradle` separately and that `Android Studio` downloads its own `gradle` anyway.